questionpy_server/misc.py

```diff
@@ -0,0 +1,24 @@
+from inspect import Parameter, signature, isclass
+from typing import Callable, List, Type, Tuple
+
+from questionpy_server.types import RouteHandler, M
+
+
+def get_parameters_of_type(function: Callable, cls: type) -> List[Parameter]:
+    output = []
+    param: Parameter
+    for _, param in signature(function).parameters.items():
+        if isclass(param.annotation) and issubclass(param.annotation, cls):
+            output.append(param)
```

Comment:
```suggestion
    for param in signature(function).parameters.values():
        if isclass(param.annotation) and issubclass(param.annotation, cls):
            output.append(param)
```

Wird der Typ von `param` ohne die annotation nicht korrekt inferred?